synthtool/languages/node.py

```diff
@@ -179,6 +180,22 @@ def fix(hide_output=False):
     shell.run(["npm", "run", "fix"], hide_output=hide_output)
 
 
+def fix_hermetic(hide_output=False):
+    """
+    Fixes the formatting in the current Node.js library. It assumes that gts
+    is already installed in a well known location on disk:
+    """
+    logger.debug("Copy eslint config")
+    shell.run(
+        ["cp", "-r", f"{_TOOLS_DIRECTORY}/node_modules", "."], hide_output=hide_output
+    )
+    logger.debug("Running fix...")
+    shell.run(
+        [f"{_TOOLS_DIRECTORY}/node_modules/.bin/gts", "fix", "src"],
+        hide_output=hide_output,
```

Comment:
If this fails, do we want the generation step to fail?  If so, add the argument `check=True`

Reply:
switched to check `True`.